langchain/agents/load_tools.py

```diff
@@ -331,18 +349,17 @@ def load_tools(
                 )
             sub_kwargs = {k: kwargs[k] for k in extra_keys}
             tool = _get_llm_tool_func(llm=llm, **sub_kwargs)
-            if callback_manager is not None:
```

Comment:
@hwchase17 do you know why we ONLY added the callback manager to a subset of tools before? This seems too specific to not be intentional but I'm not creative enough to imagine why we did this

Reply:
i dont know if there was a good reason, could have been unintentional